chirp/drivers/kguv980p.py

```diff
@@ -1011,21 +1011,27 @@ def get_memory(self, number):
 
         # Handle deleted channel quirks due to radio firmware
         # not always clearing the valid channel indicators
+        # _valid == 0x80 indicates channel was deleted thru radio menu and
+        # only part of the channel structure data is cleared by firmware
+        # _valid == 0x00 indicates the channel is Valid
+        # all other values the firmware uses for _valid: look for valid
+        # Rx and Tx freq values to determine validity
         if (_valid in MEM_INVALID or
-           (_valid != MEM_VALID) & ((_mem.rxfreq == 0xFFFFFFFF) or
-                                    _mem.rxfreq == 0x00000000)):
+           ((_valid != MEM_VALID) & ((_mem.rxfreq == 0xFFFFFFFF) or
+                                     _mem.rxfreq == 0x00000000))):
             mem.empty = True
             if _valid == 0x80:
                 LOG.debug("Ch %s was deleted by using radio menu" % number)
                 LOG.debug("Current Memory: \n%s" % _mem)
                 LOG.debug("Clearing Ch %s memory" % number)
-                _mem.set_raw("\xFF" * (_mem.size() // 8))
-                _nam.set_raw("\xFF" * (_nam.size() // 8))
```

Comment:
I just pushed this up, but for future reference, this is now available:

https://github.com/kk7ds/chirp/commit/e48bb5f1c662131e0ac5b6a5bf5670e4f9901e6e

Which could make these things:
```
_mem.fill_raw(b'\xFF')
_nam.fill_raw(b'\xFF')
```

Just FYI. I should have written that a long time ago :)

Reply:
Ok.  I will start to transition to this if it is what you prefer.  

Is there any reason to use one vs the other or should all set_raw move to the fill_rawm

Reply:
It's just much less typing and much cleaner looking when you're filling with the same byte. No need to convert existing stuff if you don't want to, but going forward new code that fills with a single byte like this should ideally use this helper.